packages/zarf-agent/zarf.yaml

```diff
@@ -9,6 +9,9 @@ constants:
 
 components:
   - name: zarf-agent
+    description: |
+      Mutating webhook that enables Kubernetes (or cloud) resources to run in airgap.
+      Directs requests for external resources to internal Zarf-managed resources.
```

Comment:
What would you recommend? @YrrepNoj

Reply:
It's defiantly tough to come up with a description for this that is decently descriptive but also simple to understand...

Perhaps something like this?
```
Mutating webhook that enables Kubernetes (or cloud) resource to run in airgap.
This enables the cluster to redirect requests for external resources (namely images and git repositories) that are vital to deploying and updating Zarf Packages to internal Zarf-defined resources.
```
 - Part of me want's to say 'control plane' instead of cluster but I think 'cluster' is more generic and potentially easier to understand.
 - I changed `zarf-managed` to `zarf-defined` b/c the agent will still redirect requests to external repos/registries that get defined via flags

Reply:
@YrrepNoj Where should we tell them it redirects to? I want to be sure that Ashton understands where that redirect is going to - and that it stays within their network